numpy/random/_generator.pyx

```diff
@@ -717,6 +717,12 @@ cdef class Generator:
         out : bytes
             String of length `length`.
 
+        Notes
+        -----
+        This function generates random integers from a discrete uniform
```

Comment:
If this is the way to describe what the function does, we'd need to specify the domain of this distribution of random integers.

But maybe this is not the way to describe what the function does, and instead this should focus on properties of the output bytes.

Reply:
I am sorry I didn't mean to write integers. I will fix it over the weekend.